src/common/helper/documentVerificationService.ts

```
@@ -0,0 +1,73 @@
+import { Repository } from 'typeorm';
+import { InjectRepository } from '@nestjs/typeorm';
+// import { issuerDid } from '../../../src/config/issuerDid';
+// Adjust the path as per your project structure
+import { type verr } from '@sunbird-rc/verification-sdk'; // Hypothetical Verifieer SDK import
+import { UserDoc } from '@entities/user_docs.entity';
+import { promises as fs } from 'fs';
+// import path from 'path';
+import * as path from 'path';
+import { EncryptionService } from './encryptionService';
+
+async function verify(): Promise<{ verifyCredential: Function }> {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)